app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::MerchantsController < Admin::BaseController
+
+  def index
+    @merchants = Merchant.all
+  end
+
+  def update
```

Comment:
Should we put the @merchant.toggle(:active?) in a conditional so that the status isn't toggled if we are updating something else?